verification/certifyKey.go

```diff
@@ -118,6 +36,14 @@ func TestCertifyKeySimulation(d TestDPEInstance, c DPEClient, t *testing.T) {
 	testCertifyKey(d, c, t, true)
 }
 
+func TestDiceTcbInfo(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testDiceTcbInfo(d, c, t, false)
+}
+
+func TestDiceTcbInfoSimulation(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testDiceTcbInfo(d, c, t, true)
+}
+
```

Comment:
Separate test case to validate DiceTcb with derived contexts in normal mode and Sim mode.